js/index.js

```diff
@@ -1,3 +1,5 @@
+import { Pawn } from "./pawn.js";
```

Comment:
You should be good to remove this import of the Pawn piece since the Board and Messages classes will be the only ones accessing the Pawn class directly.